frameworks/C++/ulib/src/fortune.h

```diff
@@ -152,7 +150,7 @@ class U_EXPORT Fortune {
 			{
 			u_put_unalignedp64(ptr,	    U_MULTICHAR_CONSTANT64('C','o','n','t','e','n','t','-'));
 			u_put_unalignedp64(ptr+8,   U_MULTICHAR_CONSTANT64('L','e','n','g','t','h',':',' '));
-			u_put_unalignedp64(ptr+16,  U_MULTICHAR_CONSTANT64('1','2','2','7','\r','\n','C','o'));
+			u_put_unalignedp64(ptr+16,  U_MULTICHAR_CONSTANT64('x','x','x','x','\r','\n','C','o'));
```

Comment:
We are working on a modification to the verification process that will ensure fortunes content lengths are not being hard coded. This will test with content lengths below 1000 and above 9999. This will fail.

Reply:
ok